src/Plugin/EntityLimit/UserLimit.php

```diff
@@ -157,6 +157,8 @@ public function validateAccountLimit(AccountInterface $account, EntityLimit $ent
       }
       return $access;
     }
+
+    return $access;
```

Comment:
@purushotamrai  - We should also remove the return statement from Line. no. 158. 👍  Nice catch.